test/rebar3_hex_integration_SUITE.erl

```diff
@@ -119,22 +121,43 @@ docs_test(Config) ->
     P = #{app => "valid", mocks => [docs]},
     {ok, #{rebar_state := State, repo := Repo}} = setup_state(P, Config),
     Command = ["docs"],
-    {ok, DocState} = test_utils:mock_command(rebar3_hex_docs, Command, Repo, State),
+    RepoConfig =  [{repos,[Repo]}],
+    {ok, DocState} = test_utils:mock_command(rebar3_hex_docs, Command, RepoConfig, State),
+    {ok, NewState} = rebar_prv_edoc:do(DocState),
+    ?assertMatch({ok, NewState}, rebar3_hex_docs:do(NewState)).
+
+docs_post_process_simple_test(Config) ->
+    P = #{app => "valid", mocks => [docs]},
+    {ok, #{rebar_state := State, repo := Repo}} = setup_state(P, Config),
+    Command = ["docs"],
+    RepoConfig =  [{repos,[Repo]}, {docs, {edoc, [{post_process, [{cmd, "echo eh?"}]}]}}],
+    {ok, DocState} = test_utils:mock_command(rebar3_hex_docs, Command, RepoConfig, State),
+    {ok, NewState} = rebar_prv_edoc:do(DocState),
+    ?assertMatch({ok, NewState}, rebar3_hex_docs:do(NewState)).
+
+docs_post_process_simple_error_test(Config) ->
```

Comment:
Le sigh, not sure about this test. It doesn't really tell us much other than "this doesn't blow up", I suppose that's valuable and inline with a lot of other tests in this suite.